articles/virtual-machines/flash-azure-monitor.md

```diff
@@ -43,6 +43,17 @@ To navigate to the [metrics explorer](/azure/azure-monitor/essentials/metrics-ge
 
 For a description of the metric's display values, see [VM availability metric](monitor-vm-reference.md#vm-availability-metric-preview).
 
+Users can split the VM availability by the 'Context' property.
+   :::image type="content" source="media/flash/context-property-filter.png" alt-text="Screenshot of the newly added Context property of VM availability Metric on Azure portal." lightbox="media/flash/context-property-filter.png" :::
+
+Note: The context for migrations (Service Healing and Live Migration) is unknown by default. We are working on making this information available in the future.
+
+The legend at the bottom indicates the color associated with the possible Context: Unknown, Platform Initiated, and User Initiated.
```

Comment:
Please remove lines 51-52. We do not recommend explaining UI elements in our documentation as it can easily change in the future and also defeats the purpose of UI -- to be self-explanatory and visually easy to follow. When it comes to Portal / UI documentation, it should only be instructional. A step-by-step on how to perform within a scenario, and that is all. Please also remove the accompanying image here and remove that image file from the media folder too.

Reply:
Removed as requested.